cmd/conf/init.go

```diff
@@ -47,7 +47,7 @@ var InitConfigDefault = InitConfig{
 	Force:                         false,
 	Url:                           "",
 	Latest:                        "",
-	LatestBase:                    "https://snapshot.arbitrum.foundation/",
+	LatestBase:                    "https://snapshot-explorer.arbitrum.io/",
```

Comment:
This is incorrect:
See: https://snapshot.arbitrum.foundation/arb1/latest-archive.txt exists.
https://snapshot-explorer.arbitrum.io/arb1/latest-archive.txt is a 404.